corehq/apps/data_cleaning/tests/test_filters.py

```diff
@@ -142,24 +146,13 @@ def test_filter_query_is_empty(self):
                 match_type=FilterMatchType.IS_EMPTY,
             )
             filtered_query = active_filter.filter_query(query)
-            expected_query = query.empty('soil_contents')
+            expected_query = query.filter(exact_case_property_text_query('soil_contents', ''))
             self.assertEqual(
                 filtered_query.es_query, expected_query.es_query,
                 msg=f"{data_type} failed to filter the query "
-                    f"properly for FilterMatchType.is_empty"
+                    f"properly for FilterMatchType.IS_EMPTY"
             )
```

Comment:
It seems like the previous version of missing and empty queries did not behave the way we expected. Do we have, or should we create, tests that verify the output of these filters, rather than the raw query? It seems like that kind of test would have been more likely to catch this issue

Reply:
i would like to yes! that's in the works for the future, but it was too complicated to set that up and take time away from the push for the deadline. But I 100% agree we should have tests on the output and it's already in my todo items.

Reply:
https://dimagi.atlassian.net/browse/SAAS-17399